pgkit/application/templates/14-recovery.conf

```diff
@@ -0,0 +1,837 @@
+# -----------------------------
```

Comment:
Are these two files the default recovery config?

Reply:
Yes, they are. I just modified the default files according to the last versions. the only part that is not consistent with the lasts version is the stats_temp_directory parameter that was deprecated in postgres v15 and thus i removed it from the config file.

Reply:
Let's check the config files with our tests. Although we have to fix them first.